ambuda/seed/lookup/page_status.py

```diff
@@ -3,6 +3,10 @@
 import ambuda.database as db
 from ambuda.seed.utils.itihasa_utils import create_db
 
+from ambuda.logger import Logger
+
+LOGGER = Logger()
```

Comment:
We shouldn't use all-caps for the name because this convention is often used for constants (not classes).

What if we just did this to simplify all imports --

- in logger.py, add the line `logger = Logger()` to the end
- in this file and other files, just run `from ambuda.logger import logger`, and then use `logger.debug`, etc.?
- this way, we are just using a single `logger` object across all files and don't need to instantiate it in each file.